### PR TITLE
task 52147 update order status

### PR DESCRIPTION
**Pull request by haltn-3346**
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/52147)

## WHAT (optional)
- admin update order status

## Evidence (Screenshot or Video)
[Screencast from 30-08-2022 16:44:34.webm](https://user-images.githubusercontent.com/109570183/187405806-9a90869f-a485-48a6-a79d-7ffe0d8ba3bf.webm)

### CODE REVIEW

File: app/views/admin/orders/show.html.erb [hunk 1]
```diff
@@ -59,6 +63,15 @@
         </table>
         <%== pagy_bootstrap_nav @pagy %>
       </div>
+      <h2><%= t ".acction" %></h2>
+      <%= link_to approve_admin_order_path(@order), method: :put, data: { confirm: ".you_sure"}, class: "btn btn-info btn-sm" do %>
+        <i class="fa fa-check"></i><%= t ".approve"%>
```
Comment (trongxt-0611) [inline app/views/admin/orders/show.html.erb]:
```suggestion
        <i class="fa fa-check"></i><%= t ".approve" %>
```

File: app/views/admin/orders/show.html.erb [hunk 2]
```diff
@@ -59,6 +63,15 @@
...
+      <% end %>
+      <%= link_to reject_admin_order_path(@order), method: :put, data: { confirm: ".confirm"}, class: "btn btn-danger btn-sm" do %>
+        <i class="fa fa-times"></i><%= t ".reject"%>
```
Comment (trongxt-0611) [inline app/views/admin/orders/show.html.erb]:
```suggestion
        <i class="fa fa-times"></i><%= t ".reject" %>
```

**Review by trongxt-0611 — COMMENTED** [review]
cmt

**Comment by haltn-3346** [comment]
ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**